Lesson7/AdditionalTask/Validator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Reflection;
+
+namespace AdditionalTask
+{
+    public class Validator
+    {
+        public (bool isValid, string name) Validate(Worker worker)
```

Comment:
tuples are great but be careful with overusing them. I usually use tuples as a return type for private methods and helper classes methods. but for main classes in the application i prefer using objects as `ValidationResult` etc. it is not a comment for this particular case in homework code, but a point for future